src/platform/assets/components/AssetFilterBar.vue

```diff
@@ -0,0 +1,107 @@
+<template>
+  <div :class="containerClasses" data-component-id="asset-filter-bar">
+    <!-- Left Side Filters -->
+    <div :class="leftSideClasses" data-component-id="asset-filter-bar-left">
+      <!-- File Format MultiSelect -->
+      <MultiSelect
+        v-model="fileFormats"
+        label="File formats"
+        :options="fileFormatOptions"
+        :class="selectClasses"
+        data-component-id="asset-filter-file-formats"
+        @update:model-value="handleFilterChange"
+      />
+
+      <!-- Base Model MultiSelect -->
+      <MultiSelect
+        v-model="baseModels"
+        label="Base models"
+        :options="baseModelOptions"
+        :class="selectClasses"
+        data-component-id="asset-filter-base-models"
+        @update:model-value="handleFilterChange"
+      />
+    </div>
+
+    <!-- Right Side Sort -->
+    <div :class="rightSideClasses" data-component-id="asset-filter-bar-right">
+      <!-- Sort SingleSelect -->
+      <SingleSelect
+        v-model="sortBy"
+        label="Sort by"
+        :options="sortOptions"
+        :class="selectClasses"
+        data-component-id="asset-filter-sort"
+        @update:model-value="handleFilterChange"
+      >
+        <template #icon>
+          <i class="icon-[lucide--arrow-up-down] size-3" />
```

Comment:
I swear, there are at least 3 entirely different ways to display an icon in this PR.